src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -441,6 +472,17 @@ public Map<String, ColumnProperties> getTableSchema() {
     return this.tableColumns;
   }
 
+  /**
+   * Get the type of channel, please referring to {@link
+   * net.snowflake.ingest.streaming.OpenChannelRequest.ChannelType} for the supported channel type
+   *
+   * @return type of the channel
+   */
+  @Override
+  public OpenChannelRequest.ChannelType getType() {
+    return OpenChannelRequest.ChannelType.CLOUD_STORAGE;
+  }
+
```

Comment:
Should we rename this class to SnowflakeStreamingIngestChannelCloud to be parallel with the rowset?

Reply:
Yes, I can do that in a separate PR